code/modules/mob/living/simple_animal/abnormality/aleph/nothing_there.dm

```diff
@@ -312,12 +313,11 @@
 	return adjusted_chance
 
 /mob/living/simple_animal/hostile/abnormality/nothing_there/work_complete(mob/living/carbon/human/user, work_type, pe, work_time)
-	. = ..()
 	worker = null
 	if(get_attribute_level(user, JUSTICE_ATTRIBUTE) < 80)
 		if(!istype(disguise)) // Not work failure
 			datum_reference.qliphoth_change(-1)
-	return
+	return ..()
```

Comment:
Don't you see comment directly above? This will screw up the logic when user fails work.